database/postgres/dal/sport.py

```diff
@@ -41,4 +41,10 @@ def delete_sport(self, sport_id: int) -> bool:
             return False
         self.db_session.delete(sport)
         self.db_session.commit()
-        return True
\ No newline at end of file
+        return True
```

Comment:
Since we are here anyway - please add empty lines. I hate it when code lines are not properly coupled